bika/lims/controlpanel/bika_sampletypes.py

```diff
@@ -5,216 +5,251 @@
 # Copyright 2018 by it's authors.
 # Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
 
-from AccessControl import ClassSecurityInfo
-from Products.ATContentTypes.content import schemata
-from Products.Archetypes import atapi
-from Products.Archetypes.ArchetypeTool import registerType
-from Products.CMFCore import permissions
-from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.utils import safe_unicode
+import collections
+import json
+
+import plone
+from bika.lims import bikaMessageFactory as _
 from bika.lims.browser import BrowserView
 from bika.lims.browser.bika_listing import BikaListingView
 from bika.lims.config import PROJECTNAME
-from bika.lims import bikaMessageFactory as _
-from bika.lims.utils import t
-from plone.app.layout.globals.interfaces import IViewView
-from bika.lims.content.bikaschema import BikaFolderSchema
 from bika.lims.interfaces import ISampleTypes
+from bika.lims.utils import get_link
 from plone.app.content.browser.interfaces import IFolderContentsView
-from plone.app.folder.folder import ATFolder, ATFolderSchema
-from zope.interface.declarations import implements
+from plone.app.folder.folder import ATFolder
+from plone.app.folder.folder import ATFolderSchema
+from plone.app.layout.globals.interfaces import IViewView
+from Products.Archetypes import atapi
+from Products.ATContentTypes.content import schemata
 from Products.CMFCore.utils import getToolByName
-import json
-import plone
+from Products.CMFPlone.utils import safe_unicode
+from zope.interface.declarations import implements
+
+
+# TODO: Separate content and view into own modules!
```

Comment:
:+1: you already did it for Methods in https://github.com/senaite/senaite.core/pull/913 . Also, this `controlpanel` package, better if we can move the views to `browser` package too.